### PR TITLE
Add Carbonite

### PR DESCRIPTION
**Pull request by 40504**
another option

</img>

### CODE REVIEW

**Comment by Carlgo11** [comment]
Odd company. They talk about how GDPR works and offer the site in German, Dutch, French and Italian yet their terms of use state the service is only meant for U.S. residents and that any legal disputes must be resolved in Boston :thinking: 

Anyways, the alternative logo is probably better as it's their favicon.

**Review by Carlgo11 — APPROVED** [review]

**Comment by phallobst** [comment]
How about this icon as seen on https://www.carbonite.com/what-is-carbonite/carbonite-webroot-opentext:

**Comment by Carlgo11** [comment]
@40504 even though the image looks really good you still need to actually provide the data as well 😉

File: entries/c/carbonite.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "Canva": {
```
Comment (phallobst) [inline entries/c/carbonite.com.json]:
```suggestion
    "Carbonite": {
```
Thou shalt not copy-and-paste! :stuck_out_tongue_winking_eye:

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/c/carbonite.com.json (hunk 1 above)
Comment (40504) [reply entries/c/carbonite.com.json]: :)

Comment (40504) [reply entries/c/carbonite.com.json]: ups

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — COMMENTED** [review]
The screenshots on the doc suggest that you can have the 2FA code sent by email as well ("Don't have access to your phone?"). Can you confirm that?

**Comment by 40504** [comment]
> The screenshots on the doc suggest that you can have the 2FA code sent by email as well ("Don't have access to your phone?"). Can you confirm that?

I can't confirm as I don't have an account there.

**Comment by 40504** [comment]
It requires a phone number and I do not want to use my own. Temporary phone numbers can't be used on Carbonite.

**Review by phallobst — APPROVED** [review]

**Pull request merged**